x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/string/Replace.java

```diff
@@ -39,6 +41,8 @@
 public class Replace extends EsqlScalarFunction {
     public static final NamedWriteableRegistry.Entry ENTRY = new NamedWriteableRegistry.Entry(Expression.class, "Replace", Replace::new);
 
+    static final long MAX_RESULT_LENGTH = MB.toBytes(1);
```

Comment:
This is based on the same Repeat logic: https://github.com/elastic/elasticsearch/blob/c990377c955ffeca7d08235411a0c468bba1ac36/x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/string/Repeat.java#L43

Reply:
nit: We could make both use the same constant, then future readers can easily see that it's being used in 2 cases which tackle a common problem.

Reply:
Moved it to `ScalarFunction`. I don't expect aggs to need this, but we could move it later